### ISSUE TITLE
Lack of Main Landmark

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#53]
**Pagina:**
[Rijkshuistijl Community Template](https://rijkshuisstijl-community-templates-git-102644-nl-design-system.vercel.app/developer-overheid/homepage)

**Probleembeschrijving:**
De webpagina mist een Main Landmark.

**Referentiecode:**
`<html class="utrecht-root">`

**Gerelateerde WCAG:**
WCAG Best Practice

**Referentieafbeelding:**

<img width="1564" alt="Homepage without a Main Landmark" src="https://github.com/user-attachments/assets/678bc784-113b-456d-851f-6699c5f05fe7" />
